maintenance_request_stage_email/models/res_config_settings.py

```diff
@@ -0,0 +1,23 @@
+# © 2021 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See https://www.gnu.org/licenses/lgpl-3.0.html
+from odoo import fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = "res.config.settings"
+
+    maintenance_request_id = fields.Many2one(
+        comodel_name="maintenance.request",
+        index=True,
+    )
+    maintenance_request_user = fields.Many2many(
+        comodel_name="res.partner",
+        related="company_id.maintenance_request_user",
+        string="Notify Contacts",
+        readonly=False
+    )
```

Comment:
Better naming and description:
```suggestion
    maintenance_request_open_user_ids = fields.Many2many(
        comodel_name="res.partner",
        related="company_id.maintenance_request_open_user_ids",
        string="Notify Contacts for open requests",
        readonly=False
    )
```